src/cfml/system/services/ServerService.cfc

```diff
@@ -1037,7 +1037,7 @@ component accessors="true" singleton {
 
 		serverInfo.trayOptions.prepend( { 'label':'Open...', 'items': openItems, "image" : expandPath('/commandbox/system/config/server-icons/open.png' ) } );
 
-		// serverInfo.trayOptions.prepend( { 'label' : 'Restart Server', 'hotkey':'R', 'action' : 'restartserver', 'image': expandPath('/commandbox/system/config/server-icons/home.png' ) } );
+		serverInfo.trayOptions.prepend( { 'label' : 'Restart Server', 'hotkey':'R', 'action' : "runAsync" , "command" : systemSettings.getSystemSetting( 'java.class.path' ) & " server restart " & "'#serverInfo.name#'", 'image': expandPath('/commandbox/system/config/server-icons/home.png' ), 'workingDirectory': defaultwebroot} );
```

Comment:
@mgmathus It's not necessary to use the jvm prop here, just use `box` and let the check below do its thing. Otherwise, you're duplicating logic.